### ISSUE TITLE
MSAL version 1.2.2 - use of 'finally' results in IE failure of silent auth

### ISSUE THREAD

**Issue by shane7218** [AzureAD/microsoft-authentication-library-for-js#1595]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [X] `msal@1.2.2`

**Important**: Please fill in your exact version number above, e.g. `msal@1.1.3`.

## Description
Use of MSAL version 1.2.2 introduced the use of a 'finally' statement in a telemetry portion of the code after a silent auth token is requested. The silent auth token call to the Authority works and MSAL saves this to the session storage, however an unexpected exception is thrown before the token is returned to the calling code. This error only occurs in Internet Explorer

## Security
No

## Regression
Yes

## Configuration
N/A

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps
1. Load Msal 1.2.2 in JS in Internet Explorer 11
2. Attempt to retrieve a silent auth token for any resource
3. Uncaught exception will be thrown from MSAL causing silent auth to fail

## Expected behavior
Silent auth should succeed and a token returned to the caller

## Browsers
IE 11 only

**Comment by jasonnutter** [comment]
@shane7218 This should be fixed in 1.3.0, can you please try that and let us know if it is fixed?

**Comment by jasonnutter** [comment]
Closing as fixed, let us know if this is still an issue.
